packages/frontend-v2/utils/course/getCourseCoreqs.ts

```diff
@@ -4,7 +4,7 @@ import { ScheduleCourse2, INEUReq, INEUReqCourse } from "@graduate/common";
 /** Get all the required(ANDs only) coreqs for a given course. */
 export async function getRequiredCourseCoreqs(
   course: ScheduleCourse2<unknown>,
-  catalogYear: number
+  catalogYear?: number
```

Comment:
Just confirming that even though we made catalog year optional, we technically still just pull the class from whatever the most recent year was right? Would this mean that the user would still see the pre-reqs and co-reqs for that since that could be misleading (though they did choose to not include that info)

Reply:
when catalog year is not provided, it is assumed that we don't care about the co-reqs and pre-reqs because we don't want to display them to the user, which is the case for transfer courses.